src/vaishnaviVJun20_arrayExamples/FirstDifferIndex.java

```diff
@@ -0,0 +1,19 @@
+package vaishnaviVJun20_arrayExamples;
+
+public class FirstDifferIndex {
+	
+	void findFirstDifferIndex(int[] array1, int[] array2) {
+	   for(int index=0;index<array1.length;index++) {
+		   if(array1[index]!= array2[index])
+			   System.out.println("Values are not matching at index->" +index);
+	   }	
+	}
+
+	public static void main(String[] args) {
+		FirstDifferIndex firstDifferIndex = new FirstDifferIndex();
+		int[] array1 = {10,2,9,14,3};
+		int[] array2 = {10,2,18,14,3};
```

Comment:
If here arrays value changed then your code is not working without return statement.

Reply:
Done